src/condor_tools/htcondor_cli/eventlog.py

```diff
@@ -68,40 +91,150 @@ def __init__(self, logger, log_file, **options):
                     event_summary["CPU Usage"] = cpu_usage_formatted
                     if event.type == htcondor.JobEventType.JOB_EVICTED:
                         event_summary["Evictions"] = event_summary.get("Evictions", 0) + 1
-                        event_summary["last_exec_time"] = datetime.fromisoformat(event.get("EventTime")).strftime("%-m/%-d %H:%M")
-
-                    event_summary["Evict Time"] = datetime.fromisoformat(event.get("EventTime")).strftime("%-m/%-d %H:%M")
+                        event_summary["last_exec_time"] = datetime.strptime(event.get("EventTime"), "%Y-%m-%dT%H:%M:%S").strftime("%-m/%-d %H:%M")
+                    event_summary["Evict Time"] = datetime.strptime(event.get("EventTime"), "%Y-%m-%dT%H:%M:%S").strftime("%-m/%-d %H:%M")
                     delta = datetime.strptime(event_summary["Evict Time"], "%m/%d %H:%M") - datetime.strptime(event_summary["Start Time"], "%m/%d %H:%M")
                     wall_time = f"{delta.days}+{delta.seconds//3600:02d}:{(delta.seconds//60)%60:02d}:{delta.seconds%60:02d}"
                     event_summary["Wall Time"] = wall_time
                     # get total execution time for calculating good time
-                    event_time = datetime.fromisoformat(event.get("EventTime")).strftime("%-m/%-d %H:%M")
+                    event_time = datetime.strptime(event.get("EventTime"), "%Y-%m-%dT%H:%M:%S").strftime("%-m/%-d %H:%M")
                     delta = datetime.strptime(event_time, "%m/%d %H:%M") - datetime.strptime(event_summary["last_exec_time"], "%m/%d %H:%M")
                     good_time = f"{delta.days}+{delta.seconds//3600:02d}:{(delta.seconds//60)%60:02d}"
                     event_summary["Good Time"] = good_time
-        
-        # determine maximum length of each column
-        max_job_id_len = max(len(str(job_id)) for job_id in event_summaries.keys())+2
-        max_host_len = max(len(data.get("Host", "")) for data in event_summaries.values())+2
-        max_start_time_len = max(len(data.get("Start Time", "")) for data in event_summaries.values())+2
-        max_evict_time_len = max(len(data.get("Evict Time", "")) for data in event_summaries.values())+2
-        max_evictions_len = len(str("Evictions"))+2
-        max_time_format = len(str("0+00:00:00"))+2
+                    # get Return Value
+                    event_summary["Return Value"] = event.get("ReturnValue")
+        # will probably need to refactor this to be more efficient for other groupby options
+        if groupby:
+            if groupby == "GLIDEIN_ResourceName":
```

Comment:
There doesn't seem to be any reason for this code to be specific to this value, except for one place where you use the constant `"Site"` instead of just using the variable `groupby` (see comment above).